app/javascript/modules/listingDetailsLottery/ListingDetailsLotteryResults.tsx

```diff
@@ -15,11 +15,11 @@ import { ListingDetailsLotterySearchForm } from "./ListingDetailsLotterySearchFo
 import { localizedFormat, renderInlineWithInnerHTML } from "../../util/languageUtil"
 import ErrorBoundary, { BoundaryScope } from "../../components/ErrorBoundary"
 
-export interface ListingDetailsLotteryProps {
+export interface ListingDetailsLotteryResultsProps {
   listing: RailsListing
 }
 
-export const ListingDetailsLottery = ({ listing }: ListingDetailsLotteryProps) => {
+export const ListingDetailsLotteryResults = ({ listing }: ListingDetailsLotteryResultsProps) => {
```

Comment:
This was renamed as I realized this component should only display if the lottery is complete. There is another lottery component that will display with lottery info